### DISCUSSION TITLE
Octernship

### DISCUSSION THREAD

**Discussion by shanksandbox** [community/community#50037] [GitHub Education]
Oh oh, looks like you need to contribute a bit more to the GitHub community before you’re eligible for an Octernship. Check out our guide on [how to contribute to open-source projects](https://opensource.guide/how-to-contribute/) to learn how you can make a meaningful impact! Keep up the great work and you’ll be a perfect fit in no time!

Why am i getting this? I have 59 repositories and avg of 10 commits per month.
How do i solve this?

**Comment by Chinmay-Deo-2004** [comment]
I'm getting this as well, though I'm not as frequent of a contributor as you, I've done quite a few commits well. I guess it's been added recently to gauge which applicants are still contributing, so if we make a few more commits, we should be in the clear. This must've been a screening criterion to filter out applicants who've been major contributors in the past but are no longer pursuing the same.

**Comment by Prathamesh-Ghatole** [comment]
I too have worked as a GSoC contributor at MetaBrainz before, but all the code for the project was hosted on my personal repo, so I barely have 2 PRs. 🥲

I have fair amount of experience working with OSS technologies, but IDK how to prove it here without PRs 🫠

**Comment by srn08** [comment]
Hey, i am getting this issue too, how to fix this?

**Comment by bonzonic** [comment]
I had access to the octernships previously and lost it this week. What are the possible causes of this?

**Reply by Mayhul-Jindal** [reply]
ya same thing with me

**Comment by DevRelMalima** [comment]
What about people that have contributed to outreachy open source program

**Comment by Mayhul-Jindal** [comment]
so I basically pushed some of my code on GitHub (mostly my personal projects which I have been working on), and that thing just automatically disappeared. Most probably they have automated the part where they check your github activity. Just keep pushing changes and do some contribution, the message will disappear automatically
